lib/eddystone-beacon-scanner.js

```diff
@@ -159,24 +166,50 @@ EddystoneBeaconScanner.prototype.parseUidData = function(data) {
 EddystoneBeaconScanner.prototype.parseUrlData = function(data) {
   return {
     txPower: data.readInt8(1),
-    url: urlDecode(data.slice(2))
+    url: urlDecode(data.slice(2)),
+    prefix: this.prefix(data.readInt8(2))
   };
 };
 
 EddystoneBeaconScanner.prototype.parseTlmData = function(data) {
   return {
-    tlm: {
-      version: data.readUInt8(1),
-      vbatt: data.readUInt16BE(2),
-      temp: data.readInt16BE(4) / 256,
-      advCnt: data.readUInt32BE(6),
-      secCnt: data.readUInt32BE(10)
-    }
+    version: data.readUInt8(1),
+    battery: data.readUInt16BE(2),
+    temperature: data.readInt16BE(4) / 256,
+    pdu: data.readUInt32BE(6),
+    time: data.readUInt32BE(10)
```

Comment:
It seems like this is going to break the external API. Is that really necessary?

Reply:
I agree, its not necessary